src/components/contacts-list/contacts-list-container.js

```diff
@@ -0,0 +1,11 @@
+import { connect } from 'react-redux';
+import { fetchContacts } from '../../actions';
+import ContactsList from './ContactsList';
+
+function mapStateToProps(state) {
+  return {
+    contacts: state.contacts
+  };
+}
+
+export default connect(mapStateToProps, { fetchContacts })(ContactsList);
```

Comment:
If you would have a long list of actions to map here (the way `fetchContacts` is now set), you would have a really long list here.
A better solution that helps with readability is to create a const mapDispatchToProps and send it as param to `connect`. Here is an example:

```
const mapDispatchToProps = dispatch => ({
  fetchContacts: () => dispatch(fetchContacts()),
});
export default connect(mapStateToProps, mapDispatchToProps)(ContactsList);
```